src/content/docs/workflows/example-workflows/pre-mfa-workflow.mdx

```diff
@@ -0,0 +1,150 @@
+---
+page_id: ca483387-5987-4449-a775-a9dfdedc0c7c
+title: Pre-MFA workflow
+sidebar:
+  order: 5
+relatedArticles:
+  - 85c135db-f55a-4edf-8d1b-0af4140c1925
+  - d672fca7-6c6a-49f8-85c9-6e86ce99f440
+---
+
+Trigger: `user:pre_mfa`
+
+This trigger is fired after the user has complete single factor authentication (e.g email + password or Google) and determined which organization (if any) they are trying to access. 
+
+## Example use cases
+
+### Determining if MFA is required based on data in an external service
+
+You may be using a service like Zanzibar or OpenFGA for fine-grained access control and wish to call out to it to determine if a user needs to complete MFA.
+
+### Making MFA required if a user has a certain permission
+
+You may wish to enforce MFA for users who have some sensitive permissions such as `delete:project` .
+
+### Skipping MFA for certain organizations
+
+You want to enforce MFA at an environment level for all organizations, but there are a few who do not want to adopt your policy.
+
+### MFA prompts
+
+When you only want to prompt MFA when a user has not been prompted for MFA over a certain time period.
+
+## Workflow code
+
+### The event object
+
+The main argument provided to your code is the Kinde workflow `event` object which has two keys `request` and `context`. This gives you access to the reason the workflow was triggered and additional relevant datapoints. Here’s an example:
+
+```jsx
+{
+  "request": {
+  },
+  "context": {
+    "auth": {
+			"connectionId": "conn_01945d3ccf4926118bfdeb6e1158edb5" // connection ID the user auth'd with
+		},
+	  "domains": {
+	    "kindeDomain": "https://example.kinde.com"
+	  },
+    "mfa": {
+      "policy": "required", // required | optional | off
+      "context": "environment", // environment or organization
+      "enabled_factors": ["mfa:sms", "mfa:email", "mfa:authenticator_app"], // factors you have enabled for this context
+      "is_user_role_exempt": false, // advanced orgs can make specific roles exempt from MFA
+      "is_user_connection_exempt": false // advanced orgs can make enterprise connections exempt from MFA
```

Comment:
@clairekinde11 my fault, all 3 of these should be camelCase 
`enabledFactors`
`isUserRoleExempt`
`isUserConnectionExempt`